src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleCloudStorageDAO.scala

```diff
@@ -307,6 +351,7 @@ class HttpGoogleCloudStorageDAO(
       .build()
   }
 
+  def adminGroupName = s"${groupsPrefix}-ADMIN@${appsDomain}"
```

Comment:
what about this? can you figure out what our different appsDomains are or make it differently configurable?